docs/tutorial-securitylist-manager-mode.md

```diff
@@ -0,0 +1,93 @@
+# Tutorial
```

Comment:
I'm not sure we need a tutorial specifically for the annotation, however, the possible annotation options and their effects certainly need to be documented front and centre.